static/js/modules/columns.js

```diff
@@ -506,6 +513,31 @@ var reports = {
   }
 };
 
+var debts = [
+  {
+    data: 'committee',
+    orderable: false,
+    className: 'all column--large',
+    render: function (data, type, row) {
+      return row.committee.name;
+    }
+  },
+  {
+    data: 'creditor_debtor_name',
+    orderable: false,
+    className: 'all column--large',
+  },
+  {
+    data: 'nature_of_debt',
+    orderable: false,
+    className: 'all column--large',
+  },
+  dateColumn({data: 'load_date', orderable: true, className: 'min-tablet hide-panel column--med'}),
```

Comment:
Not sure what value we should use for the date here, but I don't think it should be `load_date` since that's an internal value for when it was loaded into the db.